assets/stylesheets/layout/_main.scss

```diff
@@ -280,6 +280,13 @@ html.iframed {
 	line-height: 1;
 }
 
+/* Emoji and smileys */
+.emoji,
```

Comment:
Isn't there a better place to put these styles in? We also repeat almost the same code [here](https://github.com/Automattic/wp-calypso/blob/master/client/reader/post-byline/_style.scss#L9), [here](https://github.com/Automattic/wp-calypso/blob/master/client/reader/_style.scss#L13) and [here](https://github.com/Automattic/wp-calypso/blob/master/client/reader/full-post/_style.scss#L218):